### ISSUE TITLE
Wrong indent for root array

### ISSUE THREAD

**Issue by faceless7171** [looztra/yamkix#12]
Hello,

Thanks for this helpful tool!

If we have root array in the yaml it looks like indent is added to it too.
But I think it should be without any indent.

**Example:**
Input:
```yaml
---
- test1: 
  - test2: test2
```

Output:
```yaml
---
  - test1:
      - test2: test2
```

Expected:
```yaml
---
- test1:
    - test2: test2
```

Thanks in advance!

**Comment by looztra** [comment]
Thanks for the feedback!

I'm working on it but I need to arrange some unit and or integration tests, see branch [fix/0.5.0-is-a-mess](https://github.com/looztra/yamkix/tree/fix/0.5.0-is-a-mess)

**Comment by looztra** [comment]
fixed by #13 , available in [v0.5.1](https://github.com/looztra/yamkix/releases/tag/v0.5.1)
